op-node/rollup/derive/attributes.go

```diff
@@ -136,6 +136,17 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 			return nil, NewCriticalError(fmt.Errorf("failed to create depositsCompleteTx: %w", err))
 		}
 		afterForceIncludeTxs = append(afterForceIncludeTxs, depositsCompleteTx)
+
+		deps, isUpdate := ba.rollupCfg.IsDependencySetUpdate(l2Parent.Time, nextL2Time)
+		if isUpdate {
+			for _, newDependency := range deps {
+				dependencySetUpdateTx, err := AddDependencyBytes(seqNumber, l1Info, newDependency)
+				if err != nil {
+					return nil, NewCriticalError(fmt.Errorf("failed to create dependencySetUpdateTx: %w", err))
+				}
+				afterForceIncludeTxs = append(afterForceIncludeTxs, dependencySetUpdateTx)
```

Comment:
we still need to define is `afterForceIncludeTxs` is the right place for these txs